### ISSUE TITLE
How to generate label for unknown classes? 

### ISSUE THREAD

**Issue by ahsanfarooqui** [iCGY96/ARPL#14]
Suppose we pass an OOD image from the model. What output do we expect that would differentiate it from the training classes?

**Comment by sanqudui8ban** [comment]
read his code, the logits equals the property to distinguish known and unknown,  you can set a threhold by yourself.

**Comment by iCGY96** [comment]
You could refer to the following article, hope it can help you.
https://arxiv.org/abs/1610.02136
